### ISSUE TITLE
Pie Graph is not visible initially but on click gets visible

### ISSUE THREAD

**Issue by rohitkumarbhagat** [PhilJay/MPAndroidChart#3326]
In android card view, after we set the pie chart data, "no data in chart available" gets visible. On clicking the pie chart the data is loaded.

**Comment by danielglazer** [comment]
Working with 'MPAndroidChart:v2.2.4' I have the same issue only with Bar Chart.
I used this older version since I wanted x to be represented by String.

**Comment by tjhorner** [comment]
Have you tried `chart.invalidate()` where `chart` is your pie chart?

**Comment by muzeasd** [comment]
Yes. I am also getting this issue. I think this is because our data has some non-unique values. If I make all the data unique, then this works fine. On a side note, it partitions the data successfully. All it fails to do is, not color those portions.
